engine/src/exec.cpp

```diff
@@ -1457,26 +1465,26 @@ static bool MCPropertyFormatUIntList(uinteger_t *p_list, uindex_t p_count, char_
 {
     auto t_format =
         [](uinteger_t& x, MCStringRef& s) { return MCStringFormat(s, "%d", x); };
-    return MCPropertyFormatList(t_format, p_list, p_count, p_delimiter, r_string);
+    return MCPropertyFormatList(t_format, p_list, p_count, p_delimiter, false, r_string);
 }
 
 static bool MCPropertyFormatDoubleList(double *p_list, uindex_t p_count, char_t p_delimiter, MCStringRef& r_string)
 {
     auto t_format =
         [](double& x, MCStringRef& s) { return MCStringFormat(s, "%f", x); };
-    return MCPropertyFormatList(t_format, p_list, p_count, p_delimiter, r_string);
+    return MCPropertyFormatList(t_format, p_list, p_count, p_delimiter, false, r_string);
 }
 
 static bool MCPropertyFormatStringList(MCStringRef *p_list, uindex_t p_count, char_t p_delimiter, MCStringRef& r_string)
 {
-    return MCPropertyFormatList(MCStringCopy, p_list, p_count, p_delimiter, r_string);
+    return MCPropertyFormatList(MCStringCopy, p_list, p_count, p_delimiter, false, r_string);
 }
 
 static bool MCPropertyFormatPointList(MCPoint *p_list, uindex_t p_count, char_t p_delimiter, MCStringRef& r_string)
 {
     auto t_format =
-        [](MCPoint& p, MCStringRef& s) { return MCStringFormat(s, "%d,%d", p.x, p.y); };
-    return MCPropertyFormatList(t_format, p_list, p_count, p_delimiter, r_string);
+        [](MCPoint& p, MCStringRef& s) {return MCStringFormat(s, "%d,%d", p.x, p.y); };
```

Comment:
@livecodepanos I think a better fix here would be to change the custom formatter to something like:
```
{
if  (p.x == MININT2 && p.y == MININT2)
    return MCValueRetain(kMCEmptyString); 
else
    return MCStringFormat(s, "%d,%d", p.x, p.y); 
}
```